### ISSUE TITLE
Setting default options for checkbox type results in error

### ISSUE THREAD

**Issue by Devqon** [ngx-formly/ngx-formly#1836]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[x] support request
```

**Current behavior**
I am trying to override the default setting for the bootstrap checkbox type in the module declaration. This results in an formly/angular error: 'Error: No component factory found for undefined'. Not sure if I don't do it the correct way or that it is a bug.

**Expected behavior**
The default options can be overriden from the module.

**Minimal reproduction of the problem with instructions**
```
     FormlyModule.forRoot( {
      types: [
        {
          name: 'checkbox',
          defaultOptions: {
            templateOptions: {
              indeterminate: false
            }
          }
        }
      ]
    }),
```
See https://stackblitz.com/edit/ngx-formly-bootstrap-scp5pb

**What is the motivation / use case for changing the behavior?**
The bootstrap checkbox is defaulted to 'indeterminate', as seen [here](https://github.com/ngx-formly/ngx-formly/blob/198c1b559abd639b94cbe99ff3e33db4fa20d273/src/bootstrap/src/lib/types/checkbox.ts#L23). I want to configure it default as not indeterminate.

**Please tell us about your environment:**
Not applicable

* **Angular version:** 2.0.X
7.x

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X all

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
all

* **Node (for AoT issues):** `node --version` =
Not applicable

**Comment by vishnu-dev** [comment]
I checked it with material, it works.

**Comment by Devqon** [comment]
Awesomely quick, thank you!

**Comment by aitboudad** [comment]
the fix is now part of [v5.5.1](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.5.1) release!
